en/micro-integrator/docs/administer-and-observe/logs/configuring_log4j_properties.md

```diff
@@ -1,94 +1,625 @@
-# Configuring Log4j2 Properties
+# Configuring Logs
 
-All WSO2 products are shipped with the log4j2 logging capabilities, which generates administrative activities and server side logs. The `log4j2.properties` file, which governs how logging is performed by the server can be found in the `MI_HOME/conf` directory.
+## Introduction
 
-There are three main components when configuring log4j2: **Loggers**, **Appenders**, and **Layouts**. 
+All WSO2 products are shipped with log4j2 logging capabilities, which generate server side logs. The `log4j2.properties` file, which governs how logging is performed by the server can be found in the `MI_HOME/conf` directory.
 
-## Setting the log level
+??? note "Java logging and Log4j2 integration"
+    In addition to the logs from libraries that use Log4j2, all logs from libraries that use the Java logging framework are also visible in the same log files. That is, when Java logging is enabled in Carbon, only the Log4j2 appenders will write to the log files. If the Java Logging Handlers have logs, these logs will be delegated to the log events of the corresponding Log4j2 appenders. A Pub/Sub registry pattern implementation has been used in the latter mentioned scenario to plug the handlers and appenders. The following default log4j2 appenders in the `log4j2.properties` file are used for thisimplementation:
```

Comment:
A spacing issue in last line

Reply:
This is not an issue actually. Before the bullet points, we seem to require this space. Otherwise, it won't render properly.